library-app/src/components/pages/Login/Login.tsx

```diff
@@ -61,9 +61,9 @@ const Login = () => {
     } else {
       await LoginRequest(formData)
         .then(({ data }) => {
-          localStorage.setItem('accessToken', data.accessToken)
-          localStorage.setItem('refreshToken', data.refreshToken)
-          localStorage.setItem('expiration', data.expiration)
+          localStorage.setItem('accessToken', data.AccessToken)
```

Comment:
You should use separate service to work with token and remove this logic from here

Reply:
did it